src/public/parts/shells/browserIntegration/public/templates/autorun_script_table.html

```
@@ -14,7 +14,7 @@
 				<div class="font-bold">${(script.comment || '').replaceAll('<', '&lt;' )}</div>
 			</td>
 			<td>
-				<code class="p-1 bg-base-300 rounded text-sm">${script.urlRegex.replaceAll('<', '&lt;')}</code>
+				<code class="p-1 bg-base-300 rounded-field text-sm">${script.urlRegex.replaceAll('<', '&lt;')}</code>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)